tests/network/vmi_multus.go

```diff
@@ -576,6 +580,45 @@ var _ = SIGDescribe("[Serial]Multus", func() {
 			})
 		})
 
+		Context("kubectl get vmi test for primary and secondary interfaces", func() {
+			It("should verify vmi ip value match primary interface ip value", func() {
+				Expect(createBridgeNetworkAttachmentDefinition(util.NamespaceTestDefault, "linux-br", bridge10CNIType, bridge10Name, 0, "", bridge10MacSpoofCheck)).To(Succeed())
+
+				vmi := libvmi.NewCirros(libvmi.WithNetwork(v1.DefaultPodNetwork()))
+				vmi.Spec.Domain.Devices.Interfaces = []v1.Interface{*newVMISpecIfaceWithBridgeBinding("nic1"), *v1.DefaultBridgeNetworkInterface(), *newVMISpecIfaceWithBridgeBinding("nic2")}
+				vmi.Spec.Networks = append(vmi.Spec.Networks, *newVMISpecMultusNetwork("nic1", "linux-br"), *newVMISpecMultusNetwork("nic2", "linux-br"))
+
+				vmi, err = virtClient.VirtualMachineInstance(util.NamespaceTestDefault).Create(vmi)
+				Expect(err).ToNot(HaveOccurred())
+				vmi = tests.WaitForSuccessfulVMIStart(vmi)
+
+				var primaryIfaceIP string
+				for _, currStatusIface := range vmi.Status.Interfaces {
+					if currStatusIface.Name == "default" {
+						primaryIfaceIP = currStatusIface.IP
+					}
+				}
+
+				var result, errStr string
+
+				wait.PollImmediate(time.Second*1, time.Second*5, func() (bool, error) {
+					result, errStr, err = clientcmd.RunCommand(clientcmd.GetK8sCmdClient(), "get", "vmi", vmi.Name)
+					if err != nil {
+						return false, nil
+					}
+					return true, nil
+				})
+
+				Expect(err).ToNot(HaveOccurred(), errStr)
```

Comment:
`Expect(err).To(Succeed(), fmt.Sprintf("error: %s", errStr))`

Reply:
fixed